src/webcomponents/clinical/clinical-analysis-update.js

```diff
@@ -416,11 +458,13 @@ class ClinicalAnalysisUpdate extends LitElement {
                                         .priorities="${this.opencgaSession.study.internal?.configuration?.clinical?.priorities}"
                                         .multiple=${false}
                                         .classes="${this.updateParams.priority ? "updated" : ""}"
-                                        .disabled="${!!this.clinicalAnalysis?.locked}"
-                                        @filterChange="${e => {
-                                            e.detail.param = "priority.id";
-                                            this.onFieldChange(e);
-                                        }}">
+                                        .disabled="${!!this._clinicalAnalysis?.locked}"
+                                        @filterChange="${
+                                            e => {
+                                                e.detail.param = "priority.id";
+                                                this.onFieldChange(e);
+                                            }
+                                        }">
```

Comment:
Same.

Reply:
After several attempts, this is the only approach that I have found to avoid the ESLint error shown below. I am not happy to prevent ESLint from blocking the commit  by using the hook --no-verify. It might be possible to configure ESLint in a different way. I will follow your advise @jmjuanes , what is your preference for now?

Reply:
Same as above

Reply:
@gpveronica I would suggest to implement the same workaround as implemented in the `sample-update`: if we need to fix the parameter just call the `onFieldChange` and provide as the second argument the new parameter name: https://github.com/opencb/jsorolla/blob/ff0cae7b7e2d8c4c748e9b974f4cfa9ab582173c/src/webcomponents/sample/sample-update.js#L358

```
@filterChange="${e => this.onFieldChange(e, "status.id")}"
```

And then add this argument in the `onFieldChange` function: https://github.com/opencb/jsorolla/blob/ff0cae7b7e2d8c4c748e9b974f4cfa9ab582173c/src/webcomponents/sample/sample-update.js#L131-L133 

(Please remember to change the old `e.detail.param` with the new `param` variable in the calls to FormUtils, for example https://github.com/opencb/jsorolla/blob/ff0cae7b7e2d8c4c748e9b974f4cfa9ab582173c/src/webcomponents/clinical/clinical-analysis-update.js#L154 ).